nix/README.md

```diff
@@ -39,9 +39,9 @@ A description of the arguments follows:
   Provides access to the cell's blocks.\
   This is a shorthand for `inputs.cells.<cell>.<cell-block>`, where `<cell>` evaluates to the cell housing this nix file.\
   Examples:
-  - `cell.library.agda-packages` only works for code in `/cells/plutus`\
+  - `cell.library.agda-packages` only works for code in `./nix/cells/plutus`\
```

Comment:
these links now look wrong? they look like relative links but the file is in `nix` already. `/nix` could look like a link from the root, but `./nix` looks wrong. I'd just leave them relative and leave the nix bit off.

Reply:
👍 will do